sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -862,7 +891,9 @@ object Radar {
      */
     @JvmStatic
     fun startTracking(options: RadarTrackingOptions) {
+        this.logger.i("Radar.startTracking", RadarLogType.SDK_CALL)
```

Comment:
I might suggest we pass `Radar.startTracking()` or even just `startTracking()`

Preference to include the parentheses to indicate that these are function calls

Similar feedback for each of these

Reply:
👍